app/javascript/src/Navigation/utils/toggle_navigation.jsx

```diff
@@ -11,7 +11,9 @@ function hideAllToggleable (selector = allToggleableElements) {
 }
 
 function toggleElements ($elements, klass = toggleEnabledClass, state) {
-  $elements.forEach($element => $element.toggleClass(klass, state))
```

Comment:
Capybara doesn't seem to like it 😞

Reply:
This is related to https://github.com/3scale/porta/pull/110#discussion_r224072930